src/assets/img/icon-facebook.svg

```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="9432.721 3904.3 33.671 33.7">
```

Comment:
I would rather create a `assets/icons` folder and place these files there without the `icon-` prefix

Reply:
👍